scripts/generate_test_tsv.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/python
+
+import click
+import shutil
+import pandas as pd
+import mmeds.config as fig
+from mmeds.util import join_metadata, write_metadata, load_metadata
+from random import randrange, getrandbits
+from pathlib import Path
+import numpy as np
+import datetime as dt
+
+__author__ = "Adam Cantor"
+__copyright__ = "Copyright 2021, The Clemente Lab"
+__credits__ = ["Adam Cantor", "Jose Clemente"]
+__license__ = "GPL"
+
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('-p', '--path', required=True, type=click.Path(exists=True),
+              help='Path to put the created test files')
+def main(path):
+    file_path = Path(path) / 'validation_files'
+    print('Output: ' + str(file_path))
+    for test_file in file_path.glob('validate_*'):
+        print('Deleting {}'.format(test_file))
+        Path(test_file).unlink()
+
+    # Create the subjects' test files
+    sub_df = pd.read_csv(Path(path) / Path(fig.TEST_SUBJECT_SHORT).name, sep='\t', header=[0,1], na_filter=False)
+    write_error_files(sub_df, 'subject', file_path)
+
+    #Create the specimens' test files
+    spec_df = pd.read_csv(Path(path) / Path(fig.TEST_SPECIMEN_SHORT).name, sep='\t', header=[0,1], na_filter=False)
+    write_error_files(spec_df, 'specimen', file_path)
+ 
+def write_test_metadata(df, output_path, output_name):
+    """
+    Write a dataframe or dictionary to a mmeds format metadata file.
+    ================================================================
+    :df: A pandas dataframe or python dictionary formatted like mmeds metadata
+    :output_path: The path to write the metadata to
+    """
+    if isinstance(df, pd.DataFrame):
+        mmeds_meta = df.to_dict('list')
+    else:
+        mmeds_meta = df
+
+    # Create the header lines
+    lines = ['\t'.join([key[0] for key in mmeds_meta.keys()]),
+             '\t'.join([key[1] for key in mmeds_meta.keys()])]
+
+    for row in range(len(df)):
+        new_line = []
+        for item in mmeds_meta.values():
+            new_line.append(str(item[row]))
+        lines.append('\t'.join(new_line))
+    output = Path(output_path)
+    if not output.exists():
+        output.mkdir(parents=True, exist_ok=True)
+    output = Path(output_path) / Path(output_name)
+    if not output.exists():
+        output.touch()
+    output.write_text('\n'.join(lines) + '\n')
+
+
+def write_error_files(df, file_type, file_path):
+    # Contains all the headers
+    # This takes up many lines and is hard-coded, not ideal
+    if file_type == 'subject':
```

Comment:
Oh I should've mentioned. There are existing global variables you should be using here. They are contained in `mmeds/config.py`. `SPECIMEN_TABLES` and `SUBJECT_TABLES` contain a list of all the tables. TABLE_COLS is a dictionary where the values are a list of all the columns indexed by the name of the related table. Since the metadata format does get updated from time to time we want to minimize the number of locations that need to be changed.